integration_tests/test_alascan.py

```diff
@@ -0,0 +1,46 @@
+import tempfile
+from pathlib import Path
+
+import pytest
+import shutil
+
+from haddock.modules.analysis.alascan import DEFAULT_CONFIG as DEFAULT_ALASCAN_CONFIG
+from haddock.modules.analysis.alascan import HaddockModule as AlascanModule
+from haddock.libs.libontology import PDBFile
+from . import CNS_EXEC, DATA_DIR, has_cns
+
+@pytest.fixture
+def alascan_module():
+    with tempfile.TemporaryDirectory(dir=".") as tmpdir:
+        alascan = AlascanModule(
+            order=0, path=tmpdir, initial_params=DEFAULT_ALASCAN_CONFIG
+        )
+        alascan.params["int_cutoff"] = 3.5
+        alascan.params["output"] = True
+        yield alascan
+    
+class MockPreviousIO:
+    def retrieve_models(self, individualize: bool = False):
+        shutil.copy(Path("..", "tests", "golden_data", "protprot_complex_1.pdb"), Path(".", "protprot_complex_1.pdb"))
+        shutil.copy(Path("..", "tests", "golden_data", "protprot_complex_2.pdb"), Path(".", "protprot_complex_2.pdb"))
```

Comment:
Instead of `..`, add a `GOLDEN_DATA` variable to `integration_tests/__init__.py` and use it here

Reply:
good point!